### PR TITLE
feat(list-converter): a small converter who deals with column based data and do some stuff with it

### LINKED ISSUE #385: List comma separator tool
This would be a very useful tool to add to IT-Tools, I use this a lot when I do SQL work and have a list of ID's and need to put them into a format to use in  IN statement.

https://capitalizemytitle.com/tools/column-to-comma-separated-list/

### PR DESCRIPTION
**Pull request by cgoIT**
fixes #385

### CODE REVIEW

File: src/tools/list-converter/index.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import { ArrowsShuffle } from '@vicons/tabler';
```
Comment (CorentinTh) [inline src/tools/list-converter/index.ts:1]:
```suggestion
import { List } from '@vicons/tabler';
```

File: src/tools/list-converter/index.ts [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+import { defineTool } from '../tool';
+
+export const tool = defineTool({
+  name: 'List converter',
+  path: '/list-converter',
+  description:
+    'This tool can process column-based data and apply various changes (transpose, add prefix and suffix, reverse list, sort list, lowercase values, truncate values) to each row.',
+  keywords: ['list', 'converter', 'sort', 'reverse', 'prefix', 'suffix', 'lowercase', 'truncate'],
+  component: () => import('./list-converter.vue'),
+  icon: ArrowsShuffle,
```
Comment (CorentinTh) [inline src/tools/list-converter/index.ts:11]:
```suggestion
  icon: List,
```

File: src/tools/list-converter/index.ts [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+  createdAt: new Date('2023-04-28'),
```
Comment (CorentinTh) [inline src/tools/list-converter/index.ts:12]:
```suggestion
  createdAt: new Date('2023-05-07'),
```

**Review by CorentinTh — CHANGES_REQUESTED** [review]

**Review by CorentinTh — APPROVED** [review]
Thanks for this pr
Really love the richeness of the configuration

**Comment by CorentinTh** [comment]
@cgoIT I slightly updated the ui of the config section 
image

**Comment by cgoIT** [comment]
👍 very nice! Thx!

**Pull request merged**